### DISCUSSION TITLE
Installation Delphi 7

### DISCUSSION THREAD

**Discussion by MarioFasching** [graphics32/graphics32#316] [General] [answered]
The current master version does not contain installation packages for Delphi 7. How do you install this version under Delphi 7?

**Comment by andersmelander** [comment, marked as answer]
The last version to support pre-XE2 versions was 2.0: https://github.com/graphics32/graphics32/releases/tag/Graphics32_2.0_Final

**Reply by MarioFasching** [reply]
Delphi 7 is not inclouded. Packages start with 2007. Is there an earlier Version available? THX

**Reply by andersmelander** [reply]
https://github.com/graphics32/graphics32/blob/release/2.0/Source/Packages/GR32_D7.dpk
